FORMAT_ADIOS.md

```diff
@@ -32,3 +32,28 @@ Output from `bpls -A` for a boolean attribute `pybool` stored in the location of
 
 There is no convention yet for a unique representation of ADIOS2 variables with boolean type.
 Thus, implementations should cast the data to and from `unsigned char` instead.
+
+## `stepBased` Encoding of Iterations
+
+The `iterationEncoding` mode `stepBased` must be implemented via ADIOS steps.
+
+## Datasets
+
+An openPMD **data set** is represented by an group prefix that contains an ADIOS variable `__data__`.
+
+**attributes** are defined further below and can also appear at the dataset's **group** prefix level.
+
+## Attributes
+
+openPMD **attributes** stored as ADIOS `Variables` at the location where they would usually be stored.
```

Comment:
@franzpoeschel raised this is currently implemented that way and I think since we don't need to change the type of attributes over time, we can keep it so:
```suggestion
openPMD **attributes** stored as ADIOS `Variables` at the location where they would usually be stored.
The `__is_boolean__` qualifier is still stored as ADIOS `Attribute`.
```

Reply:
> openPMD **attributes** stored as ADIOS `Variables`

Is this outdated? We will remove the new ADIOS2 schema where this happens